data/json/mapgen/s_fabricstore.json

```
@@ -0,0 +1,91 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "s_fabricstore" ],
+    "//": "Frabic Outlet",
+    "weight": 100,
+    "object": {
```

```suggestion
    "object": {
```
A map's "weight" field defaults to 1000 if undefined and is functional when there are multiple OMTs with the same id